### DISCUSSION TITLE
虚拟机迁移之后， 暂时无权访问，1pctl user-info 显示不了具体的信息如何解决

### DISCUSSION THREAD

**Discussion by fpd3210** [1Panel-dev/1Panel#5804] [Q&A]
具体如下图
![image](https://github.com/user-attachments/assets/e4b91dcc-1193-4570-ab5f-1c8e3b932fdd)
![Uploading image.png…]()

**Comment by fpd3210** [comment]
补充图片 1pctl user-info 图片
![image](https://github.com/user-attachments/assets/98149636-db23-4117-a2e6-969820f726b0)

**Comment by fpd3210** [comment]
虚拟机迁移方式是直接打包其中一个虚拟机文件，然后复制到另外一台电脑上面

**Reply by wanghe-fit2cloud** [reply]
可以考虑手动执行 `1pctl update password` 命令重置密码。
